main/main.ino

```diff
@@ -29,22 +27,33 @@ float ball_distance = 0;
 float goal_angle = 0;
 bool ball_found = false;
 double ponderated_angle = 0;
-double angle_degrees = 0; 
+double angle_degrees = 0;
 float goal_distance = 0;
 double last_time = 0;
 double current_time = 0;
+double last_ball_angle = 0;
 int goal_threshold = 15;
 const int FRAME_HEIGHT = 104;
 const int FRAME_WIDTH = 158;
 const int FRAME_ROBOT = 20;
 const int FRAME_CIRCLE = 150;
+const unsigned long period = 100;
+const int delay_time = 3000;
+const int min_speed = 1000;
+const int mid_speed = 1500;
+const int max_speed = 2000;
+const int esc_pin = 6;
+int photo_value = analogRead(A2);
+int photo_value1 = analogRead(A7);
+int photo_value2 = analogRead(A3);
+int photo_value3 = analogRead(A8);
+int photo_value4 = analogRead(A9);
 
 Pixy2SPI_SS pixy;
 BNO055 my_bno;
 Servo esc;
```

Comment:
Is this `Servo` the dribbler? Use a more descriptive name

Reply:
Yes that is correct, we still need to refactor that logic.